.deploy/api/Dockerfile

```
@@ -333,6 +335,7 @@ RUN cp -r ./packages/plugins/job-search/node_modules ./node_modules/@gauzy/plugi
 RUN cp -r ./packages/plugins/knowledge-base/node_modules ./node_modules/@gauzy/plugin-knowledge-base/
 RUN cp -r ./packages/plugins/product-reviews/node_modules ./node_modules/@gauzy/plugin-product-reviews/
 RUN cp -r ./packages/plugins/sentry-tracing/node_modules ./node_modules/@gauzy/plugin-sentry/
+RUN cp -r ./packages/plugins/posthog/node_modules ./node_modules/@gauzy/plugin-posthog/
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)